### ISSUE TITLE
SDL 2.0.10

### ISSUE THREAD

**Issue by TerensTare** [veandco/go-sdl2#439]
Does this support SDL 2.0.10?

**Comment by veeableful** [comment]
Hi @TerensTare, yes! Though it seems like it's written in C++ which is incompatible with cgo.

**Comment by TerensTare** [comment]
What if I make it work in GoLang?
Maybe by rewriting it in pure Go?

**Comment by veeableful** [comment]
That sounds like a good approach. There doesn't seem to be any problem other than the amount of work you have to put into it. If you are driven to do this, I'd say go for it!

**Comment by TerensTare** [comment]
I will leave commenting up to you.
So you can write it in your own style.

**Comment by TerensTare** [comment]
There is a small problem.
The library uses SDL_gfx and I am on Windows. 
Is there any way to use SDL_gfx on WIndows?

**Comment by veeableful** [comment]
Yes, I was able to get it running [here](https://github.com/veandco/go-sdl2/issues/427#issuecomment-531489290). It seems like you just need to copy the header and .dll files to the MinGW's include and lib directories. Let me know if you have any difficulty.

**Comment by TerensTare** [comment]
It worked. Thanks a lot.
Talk to you later.

**Comment by veeableful** [comment]
No problem. I'm glad it helped :relaxed:

**Comment by TerensTare** [comment]
Hey @veeableful . I found [this](http://www.stroustrup.com/bs_faq2.html) faq hosted by Bjarne Stroustrup (the creator of C++). It says that you can export C++ classes to C code. That means that I can use C bindings to generate GoLang bindings (way simpler than rewriting the whole thing in GoLang).

**Comment by veeableful** [comment]
That's true. You could contribute to that project by writing C binding for them!
